tools/src/main/java/org/apache/kafka/tools/consumergroup/ConsumerGroupCommandOptions.java

```diff
@@ -0,0 +1,263 @@
+/*
```

Comment:
Is the tools/consumergroup a new folder? I wonder if there is a name that is more consistent with the other folders.

Reply:
> Is the tools/consumergroup a new folder?

yes.

> I wonder if there is a name that is more consistent with the other folders.

Couldn't come up with the better naming :) 
Do you have one in mind?

Reply:
I wasn't sure if we wanted to use the same convention of the other java consumer group files. But maybe that is confusing. They follow a pattern of coordinator/group. Not sure if we really have reusability amongst coordinator tools though. 

I think when we have had two word folders we usually put a dash between them (ie consumer-group)

Reply:
> I think when we have had two word folders we usually put a dash between them (ie consumer-group)

Dash can't be used in java package name.

> They follow a pattern of coordinator/group

We can move classes to `org.apache.kafka.tools.consumer.group` package.
What do you think?

Reply:
That works for me. Thanks!

Reply:
package renamed.